includes/wikia/GlobalFunctions.php

```diff
@@ -937,7 +937,11 @@ function wfTimeFormatAgo( $stamp, $hideCurrentYear = true ) {
 		$format = $wgLang->getDateFormatString( 'date', 'default' );
 		if ( $hideCurrentYear ) {
 			$format = trim( $format, ' ,yY' );
+
+			// For Japanese and Chinese the 年 character before the month number must also be stripped. See JPN-549
+			$format = str_replace('年', '', $format);
```

Comment:
Oh, I did not see it's already in Global Functions. Right now this string replacement will be done for every language, not only Japanese or Chinese. Maybe you can investigate usage of date formats in Messages*.php files like here https://github.com/Wikia/app/blob/dev/languages/messages/MessagesZh_hans.php#L239